### PR TITLE
16.0 mig auth api key

### PR DESCRIPTION
**Pull request by benoit-bertholon**

### CODE REVIEW

File: auth_api_key/models/auth_api_key.py [hunk 1]
```diff
@@ -49,11 +49,12 @@ def _clear_key_cache(self):
...
         self._retrieve_uid_from_api_key.clear_cache(self.env[self._name])
 
-    @api.model
-    def create(self, vals):
-        record = super(AuthApiKey, self).create(vals)
-        if "key" in vals or "user_id" in vals:
-            self._clear_key_cache()
+    @api.model_create_multi
+    def create(self, vals_list):
+        record = super(AuthApiKey, self).create(vals_list)
```
Comment (StefanRijnhart) [inline auth_api_key/models/auth_api_key.py]: `records`

**Comment by StefanRijnhart** [comment]
Looking good now! Please squash your migration commits.

**Comment by benoit-bertholon** [comment]
@StefanRijnhart  squashed :)

**Review by StefanRijnhart — APPROVED** [review]
Thanks!

**Comment by benoit-bertholon** [comment]
@StefanRijnhart sorry I pushed the wrong button ..

**Review by rvalyi — APPROVED** [review]

**Review by StefanRijnhart — APPROVED** [review]
No problem!

**Pull request merged**